core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/Runner.scala

```diff
@@ -407,14 +410,39 @@ object PodsConfig {
    * Currently only dealing with secrets inside volumes
    * Not taking into acccount other type of sources
    */
+  /**
+   * Currently only dealing with secrets inside volumes
+   * Not taking into acccount other type of sources
+   *
+   * How to make this optional?
+   */
   implicit val sourceConfReader: ValueReader[Volume.Source] = ValueReader.relative { config =>
-    config.as[Volume.Secret]("secret")
+    println(config)
+    val c: Config                        = config
+    val m: java.util.Map[String, AnyRef] = c.root().unwrapped()
```

Comment:
why you need unwrapped if you are only using the keyset?

Reply:
done